helm-charts/basehub/values.yaml

```diff
@@ -509,6 +513,10 @@ jupyterhub:
 
             return allowed_profiles
 
-        # Customize list of profiles dynamically, rather than override options form.
-        # This is more secure, as users can't override the options available to them via the hub API
-        c.KubeSpawner.profile_list = custom_profile_list
+        # Only set this customized profile_list *if* we already have a profile_list set
+        # otherwise, we'll show users a blank server options form and they won't be able to
+        # start their server
+        if c.KubeSpawner.profile_list:
+          # Customize list of profiles dynamically, rather than override options form.
+          # This is more secure, as users can't override the options available to them via the hub API
+          c.KubeSpawner.profile_list = custom_profile_list
```

Comment:
```suggestion
            # Customize list of profiles dynamically, rather than override options form.
            # This is more secure, as users can't override the options available to them via the hub API
            c.KubeSpawner.profile_list = custom_profile_list
```